<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyAwardParam</class>
 <widget class="QWidget" name="DailyAwardParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_common">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>一般类别</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="checkBox_vip_signin">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>VIP签到</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_daily_signin">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>每日签到</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_free_wish">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>免费许愿</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_bottom_quest">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>领取底部任务中的日常任务。会自动收起前三种任务类别</string>
      </property>
      <property name="text">
       <string>底部任务</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_tarot_treasure">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>没有免费次数时不会翻牌</string>
      </property>
      <property name="text">
       <string>塔罗寻宝</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_campsite_key">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>营地钥匙</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_common">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_cycle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>周期类别</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_cycle" rowstretch="1,1,1,1" columnstretch="3,2,3,2,0,4,1,2,3">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="5">
     <widget class="QCheckBox" name="checkBox_union_garden">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>公会花园</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_give_flowers">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>赠送鲜花</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_union_quest">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>公会任务</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QComboBox" name="comboBox_use_coupon_times">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>使用礼券赠送鲜花的次数</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="7" colspan="2">
     <widget class="QCheckBox" name="checkBox_force_destiny_tree">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>勾选后，非周一也会执行</string>
      </property>
      <property name="text">
       <string>其他日期强制领取</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="Line" name="line_cycle">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_pharaoh_treasure">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>没有免费次数时不会翻牌</string>
      </property>
      <property name="text">
       <string>法老宝藏</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_pharaoh_flop_pos">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>从左往右、从上往下的顺序</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_plant_type">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>当本公会的公会树需要种植时生效</string>
      </property>
      <property name="text">
       <string>树苗等级：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QCheckBox" name="checkBox_open_food_contest">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>打开美食大赛界面，以完成累计登录20天的任务。没找到时自动跳过</string>
      </property>
      <property name="text">
       <string>打开美食大赛界面</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QCheckBox" name="checkBox_need_fertilize">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>施肥</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_pharaoh_flop_pos">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>翻牌位置：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_use_coupon_times">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>礼券次数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QCheckBox" name="checkBox_destiny_tree">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>每周一缘分树奖励</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QCheckBox" name="checkBox_use_coupon">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>使用礼券</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_flowers_receiver">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>选择赠送对象昵称截图。该截图与房间内组队邀请截图通用</string>
      </property>
      <property name="text">
       <string>赠送对象：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_flowers_receiver">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>选择赠送对象昵称截图。该截图与房间内组队邀请截图通用</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_flowers_receiver">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>选择赠送对象昵称截图。该截图与房间内组队邀请截图通用</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QComboBox" name="comboBox_pharaoh_flop_pos_2">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>无种植权限时不要选择其他，否则可能造成不确定的影响</string>
      </property>
      <item>
       <property name="text">
        <string>无权限</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>初级公会树</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中级公会树</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>高级公会树</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_team_magic_tower">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>每周一双人魔塔奖励</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QCheckBox" name="checkBox_release_quest">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>有权发布会长任务</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QCheckBox" name="checkBox_open_backpack">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>上线后，需要打开背包才能激活情侣任务</string>
      </property>
      <property name="text">
       <string>打开背包</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QCheckBox" name="checkBox_force_team_magic_tower">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>勾选后，非周一也会执行</string>
      </property>
      <property name="text">
       <string>其他日期强制领取</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
